### ISSUE TITLE
page request: gitk

### ISSUE THREAD

**Issue by kevin91liu** [tldr-pages/tldr#2018]
I haven't used it before, but here's a link to the docs:
https://git-scm.com/docs/gitk

**Comment by agnivade** [comment]
https://github.com/tldr-pages/tldr/blob/master/pages/linux/gitk.md

**Comment by kevin91liu** [comment]
I'm on OSX.

```
MacBook-Pro-(5):~ kevin.liu $ tldr gitk
Page not found. Updating cache ..
Creating index...
Done
Page not found.
Feel free to send a pull request to: https://github.com/tldr-pages/tldr
```

Maybe this is because `gitk.md` only exists in the `linux` directory and should instead be put in the [common](https://github.com/tldr-pages/tldr/tree/master/pages/common) directory?

**Comment by agnivade** [comment]
You have to pass the os flag to get an os specific page - `tldr --os=linux gitk`. 

Can you send a PR moving the file to common ?

**Comment by kevin91liu** [comment]
Sure: https://github.com/tldr-pages/tldr/pull/2019

**Comment by pxgamer** [comment]
Closed in #2019. Thanks @kevin91liu.
